Homework_1.py

```diff
@@ -0,0 +1,97 @@
+# Задача 2: Найдите сумму цифр трехзначного числа. 
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+'''
+num = int(input('Введите 3-х значное число: '))
+
+if num <100 or num > 999:
+    print('Число не 3-х значное')
+else:
+    print((num % 10) + ((num % 100) // 10) + (num // 100))
+ '''   
+
+
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе
+# они сделали S журавликов. Сколько журавликов сделал каждый
+# ребенок, если известно, что Петя и Сережа сделали одинаковое
+# количество журавликов, а Катя сделала в два раза больше журавликов,
+# чем Петя и Сережа вместе?
+# 6 -> 1 4 1
+# 24 -> 4 16 4
+# 60 -> 10 40 10
+
+'''
+amount = int(input('Input amount of products: '))
+b = int(amount/ 3 * 2)
+a = int(b/4)
```

Comment:
Не обязательно использовать функцию `int`.
```suggestion
b = amount // 3 * 2
a = b // 4
```